Godeps

```diff
@@ -0,0 +1,7 @@
+#
+# Project Dependencies
+#
+
+github.com/gorilla/mux              master
```

Comment:
You can also pass in commit refs - if you use master you could get breaking changes form build to build.